public/stylesheet.css

```diff
@@ -0,0 +1,40 @@
+/* Includes just woff2 for Chromatic testing */
```

Comment:
Chromatic has recently gotten brittle, e.g. roughly 5 specs fail visual regression testing because the fonts are not loading before the Snapshots. The Chromatic GUI suggested moving these external assets into Storybook for better performance.

Here's an example of a failure: https://www.chromatic.com/build?appId=5f6e46f1e7081000221523cf&number=1700

And here's an example of the messaging: https://www.chromatic.com/test?appId=5f6e46f1e7081000221523cf&id=624b9c28d0d7ba003a869676

<img width="1266" alt="Screen Shot 2022-04-04 at 8 18 33 PM" src="https://user-images.githubusercontent.com/13544620/161672233-13105af9-dcf6-4a20-9a83-c99d6e647002.png">